src/components/subscribe/subscribe-form.js

```diff
@@ -11,7 +11,9 @@ import { Button } from "../button/button";
 
 export const SubscribeForm = ({ compact }) => {
   const SubscribeButton = () => (
-    <Button inputType="submit">{Wording.BUTTON}</Button>
+    <Button inputType="submit" onClick={() => void 0}>
```

Comment:
Oh. My bad. I asked for this. It was actually another component that had this removed and I just messed them in my mind.

Reply:
I guess in the future we can make this callback optional in the `Button` component so we don't have to "fix" it this way in different places

Reply:
No problem. It happens when you do lots of things at once.